androidtest/src/main/java/com/tozny/e3dbtest/KPLockScreenFragment.java

```diff
@@ -55,7 +55,7 @@ public String configName() {
 
     @Override
     public KeyAuthentication keyProtection() {
-        return KeyAuthentication.withLockScreen();
+        return KeyAuthentication.withLockScreen(5);
```

Comment:
Should we document what the value of `5` is doing?

Reply:
I don't think it's necessary - the IDE should show docs, or they can jump to source easily. In any case this is just the test app.

Reply:
:+1: